### PR TITLE
Fix stage indication for 25VNA8/24VNA9 equipment

### PR DESCRIPTION
**Pull request by mamarley**
When dehumidifying, the opstat is "dehumidify", not the actual
stage of the equipment.  However, it can be confirmed by checking
the service interface on the thermostat that dehumidification takes
place in stage 1, so indicate it as such on the dial.

### CODE REVIEW

File: public/app/views/main.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h4>Global</h4>
 		<ng-dial-gauge ng-if="status.cfgem[0]=='C'" title="Outside" units="&deg;" border-width="0" bar-color="#0000FF" bar-color-end="#FF2010" bar-width="10" ng-model="((carbus.outsideTemp || status.oat[0])-32)*5/9"></ng-dial-gauge>
 	</div>
 	<div class="col-xs-6 col-sm-4 col-md-2" ng-if="status.odu[0].type[0].includes('proteus')">
-			<ng-dial-gauge title="HP Stage" border-width="0" bar-color="#00FF00" bar-color-end="#FF0000" scale-min="0" scale-max="5" bar-width="10" ng-model="status.odu[0].opstat[0] === 'off' ? 0 : status.odu[0].opstat[0].replace('Stage ','')"></ng-dial-gauge>
+			<ng-dial-gauge title="HP Stage" border-width="0" bar-color="#00FF00" bar-color-end="#FF0000" scale-min="0" scale-max="5" bar-width="10" ng-model="status.odu[0].opstat[0] === 'off' ? 0 : status.odu[0].opstat[0].replace('Stage ','').replace('dehumidify', '2')"></ng-dial-gauge>
```
Comment (nebulous) [inline public/app/views/main.html:19]: 👍

**Comment by nebulous** [comment]
This looks fairly straightforward. Did you mean for it to still be a Draft PR?

**Comment by mamarley** [comment]
Yes, I just need to confirm that the compressor is actually in the second stage.  I didn't have physical access to the thermostat when I made the PR and it hasn't been warm enough for dehumidification since.  I should be able to get the data I need within the next couple of days though.

**Comment by mamarley** [comment]
I thought for sure that with a 4-ton condensing unit and a 5-ton coil running at ~500CFM on a system that is nominally 350CFM/ton that it must have been in stage 2, but comparing the compressor RPM when dehumidifying to the compressor RPM when running in stage 1 and 2 in the "check out" installer menu, it is definitely in stage 1.

**Pull request merged**